spec/models/article_spec.rb

```diff
@@ -4,6 +4,8 @@
   describe 'DB columns' do
     it { is_expected.to have_db_column :title }
     it { is_expected.to have_db_column :content }
+    it { is_expected.to have_db_column :longitude }
+    it { is_expected.to have_db_column :latitude }
```

Comment:
we can test this better. 
* Address is missing

User_spec is missing lat, lng and adress!